### ISSUE TITLE
ValueError: failed to parse CPython sys.version: '2.7.13

### ISSUE THREAD

**Issue by docian** [conda/conda#4901]
i've installed Anaconda on Windows but nothing works. I cannot import pandas and when I'm trying to launch conda i'm getting:
Traceback (most recent call last):
  File "C:\Users\docian\AppData\Local\Continuum\Anaconda2\Scripts\conda-script.py", line 5, in <module>
    sys.exit(conda.cli.main())
  File "C:\Users\docian\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda\cli\main.py", line 164, in main
    return conda_exception_handler(_main, *args)
  File "C:\Users\docian\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda\exceptions.py", line 589, in conda_exception_handler
    print_unexpected_error_message(e)
  File "C:\Users\docian\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda\exceptions.py", line 543, in print_unexpected_error_messag
e
    info_stdout, info_stderr = get_info()
  File "C:\Users\docian\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda\exceptions.py", line 513, in get_info
    args.func(args, p)
  File "C:\Users\docian\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda\cli\main_info.py", line 148, in execute
    from conda.api import get_index
  File "C:\Users\docian\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda\api.py", line 1, in <module>
    from .core.index import get_index
  File "C:\Users\docian\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda\core\index.py", line 8, in <module>
    from .package_cache import PackageCache
  File "C:\Users\docian\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda\core\package_cache.py", line 10, in <module>
    from .path_actions import CacheUrlAction, ExtractPackageAction
  File "C:\Users\docian\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda\core\path_actions.py", line 31, in <module>
    from ..gateways.download import download
  File "C:\Users\docian\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda\gateways\download.py", line 16, in <module>
    from ..connection import CondaSession
  File "C:\Users\docian\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda\connection.py", line 55, in <module>
    python=platform.python_implementation(),
  File "C:\Python27\Lib\platform.py", line 1498, in python_implementation
    return _sys_version()[0]
  File "C:\Python27\Lib\platform.py", line 1463, in _sys_version
    repr(sys_version))
ValueError: failed to parse CPython sys.version: '2.7.13 |Anaconda 4.3.1 (64-bit)| (default, Dec 19 2016, 13:29:36) [MSC v.1500 64 bit (AMD6
4)]'

**Comment by kalefranz** [comment]
This parse error is coming from within the python std lib.  From

    import platform
    platform.python_implementation()

Could just be corrupted python.  Maybe something @ilanschnell wants to be aware of though.

**Comment by mingwandroid** [comment]
Here you have two different Python installations and they are stepping on each other's toes.

I am guessing you elected to allow the installer to add itself to your system `PATH` but already had 'normal' Python 2.7 installed to the default location and also present in your `PATH`?

Unfortunately our installer adds its entries to the end of `PATH` so DLLs for other software frequently get loaded instead of ours (Python itself consists of many DLLs).

I think, if the Anaconda Installer is going to add to `PATH` it should add its entries to the front of `PATH` like nearly all other installers do, but I more strongly think that software should not add itself to `PATH` by default (or recommend that to users).

If you use the `Anaconda Prompt` to launch your command prompt then that will add the correct entries to the front of your `PATH` temporarily (i.e. for that command prompt instance only).

**Comment by kalefranz** [comment]
Duplicate of https://github.com/conda/conda/issues/3606

**Comment by naught101** [comment]
It's really hard to follow this chain of issues and figure out which one is active, but this causes a bunch of errors that can be really annoying, including

https://github.com/Valloric/YouCompleteMe/issues/1652

and 

https://stackoverflow.com/questions/34145861/valueerror-failed-to-parse-cpython-sys-version-after-using-conda-command

There is a suggestion at the second link that it may be due to the `|` in the version, though I haven't investigated that.

**Comment by naught101** [comment]
Actually, this may have been a conda-forge issue, my mistake.
